src/common/IOSLauncher.ts

```diff
@@ -79,9 +80,21 @@ export class IOSLauncher {
                 return IOSUtils.waitUntilDeviceIsReady(deviceUDID);
             })
             .then(() => {
+                const address =
```

Comment:
Minor: can we roll these two variable settings under the same `if/else` block of the `PreviewUtils.isTargetingBrowser(targetApp) || (appConfig && appConfig.preview_server_enabled === true)` condition? I think the intent of the functionality would read better.

Reply:
Actually never mind. It's pretty much a push. Maybe instead we can initialize an `usePreviewServer` var to hold the condition, and then use that in the ternary operations instead? It's just a bit wordy, as is.

Reply:
good suggestion, will do